main.js

```diff
@@ -32,6 +41,15 @@ export default class DesktopAppBanner {
 		if (!this.emailButton.disabled) {
 			this.emailButton.disabled = 'disabled';
 
+			document.body.dispatchEvent(new CustomEvent('oTracking.event', {
+				detail: {
+					category: 'form',
+					action: 'submit',
+					messaging: 'desktop-app-banner-form-submit'
```

Comment:
Was the idea not to have the messaging attribute consistent across all of the events for a given component? e.g 'desktop-app-banner' will probably be sufficient everywhere?